docs/develop/sdks/java/index.md

```diff
@@ -25,7 +25,7 @@ The source code can be found on GitHub: [momentohq/client-sdk-java](https://gith
 ## Resources
 
 - [Java SDK Cheat Sheet](./cheat-sheet.mdx)
-- [Java SDK Examples](https://github.com/momentohq/client-sdk-java/blob/main/examples/README.md): working example projects that illustrate how to use the Java SDK
+- [Java SDK Examples](https://github.com/momentohq/client-sdk-java/blob/main/examples): working example projects that illustrate how to use the Java SDK
```

Comment:
uh-oh @pratik151192 I think this came from your recent restructuring of the examples dir.

I think rather than change this here, I would like to add a README.md in the java repo that explains what each of the subdirs have in them and links to them. WDYT?

Reply:
ah yeah that makes sense. I can fast-follow on that but let's merge this anyway so that clients don't see a 404

Reply:
That’s fine with me but I wanted a quick fix to show Goldsky we are reacting to feedback - hopefully your suggestion can be implemented fast too?

Reply:
okay that's fine, but would like to change this back to link to your new README as soon as you add it.